src/batch/src/executor/join/hash_join.rs

```diff
@@ -2004,7 +2005,7 @@ mod tests {
                 None
             };
 
-            Box::new(HashJoinExecutor::<Key32>::new(
+            Box::new(HashJoinExecutor::<Key32<StackNullBitmap>>::new(
```

Comment:
Would you please add the default value for generic like `<B = HeapNullBitmap>` and check if we can omit to specify here (for code simplify in tests)?